src/containers/AdminCampaignEdit/components/StartCampaignButton.tsx

```diff
@@ -53,19 +83,27 @@ export const StartCampaignButton: React.FC<StartCampaignButtonProps> = (
     : "Start this campaign";
 
   return (
-    <Tooltip title={tooltipText} placement="top">
-      <span>
-        <Button
-          variant="contained"
-          color="primary"
-          style={{ width: 210 }}
-          disabled={disabled}
-          onClick={handleClick}
-        >
-          {startText}
-        </Button>
-      </span>
-    </Tooltip>
+    <>
+      <Tooltip title={tooltipText} placement="top">
+        <span>
+          <Button
+            variant="contained"
+            color="primary"
+            style={{ width: 210 }}
+            disabled={disabled}
+            onClick={handleClick}
+          >
+            {startText}
+          </Button>
+        </span>
+      </Tooltip>
+      <CampaignSurveyResponseWarningDialog
+        open={showConfirmDialog}
+        campaignId={campaignId}
+        confirmWarningAndStartCampaign={confirmWarningAndStartCampaign}
+        setShowConfirmDialog={setShowConfirmDialog}
```

Comment:
Nit: Renaming these props in the child component follows a loose convention that I prefer for readability in both the parent and child as seen here
```suggestion
        onConfirm={confirmWarningAndStartCampaign}
        onClose={setShowConfirmDialog}
```

Reply:
And we also have the functions being `handleConfirmWarningAndStartCampaign`, so we know it's a handler.